Makefile

```diff
@@ -39,3 +39,29 @@ sonar:
 .PHONY: sonar-pr-analysis
 sonar-pr-analysis:
 	mvn sonar:sonar -P sonar-pr-analysis
+
```

Comment:
This does not belong here.
Please familiarise your self with the new depcheck functionality - see [README of dep-check-suppressions](https://github.com/companieshouse/dependency-check-suppressions/).

You should just be able to run depcheck in your repo.